src/vs/editor/contrib/inlineCompletions/browser/inlineSuggestionHintsWidget.css

```diff
@@ -34,3 +34,7 @@
 	min-width: 19px;
 	justify-content: center;
 }
+
+.inlineSuggestionStatusBarItemLabel {
```

Comment:
Best to add `.monaco-editor` because otherwise embedder of the monaco/standalone editor won't like you